macros/sfdc_formula_view.sql

```diff
@@ -1,4 +1,4 @@
-{%- macro sfdc_formula_view(source_table, source_name='salesforce', reserved_table_name=source_table, fields_to_include=none, full_statement_version=false, materialization='view') -%}
+{%- macro sfdc_formula_view(source_table, source_name='salesforce', reserved_table_name=source_table, fields_to_include=none, full_statement_version=false, materialization='view', using_quoted_identifiers=false) -%}
```

Comment:
More my own curiosity: Checking the readme, these are arguments that the customer will modify depending on their own warehouse/schema variables and fields?

Reply:
Yeah exactly! And a few other non warehouse/schema specific items such as what type of version of the macro they want to use.